src/confluent_kafka/src/confluent_kafka.c

```diff
@@ -1949,6 +1990,19 @@ rd_kafka_conf_t *common_conf_setup (rd_kafka_type_t ktype,
                         Py_XDECREF(ks8);
                         Py_DECREF(ks);
                         continue;
+                } else if (!strcmp(k, "oauth_cb")) {
+                        if (h->oauth_cb) {
```

Comment:
add a PyCallable_Check here, see stats_cb above.

Reply:
Done. I have also updated my branch to the current master